### PR TITLE
Add more details to security disclosure page

### PR DESCRIPTION
**Pull request by howardjohn**
Attempt to get people to stop publicly filing security vulns in github

### CODE REVIEW

**Comment by howardjohn** [comment]
https://github.com/istio/istio/pull/27627 is related

File: content/en/about/security-vulnerabilities/index.md [hunk 1]
```diff
@@ -25,6 +25,11 @@ Send us a report whenever you:
 - Think a vulnerability is present in another project that Istio
 depends on. For example, Envoy, Docker, or Kubernetes.
 
+When in doubt, please disclose privately. This includes, but is not limited to:
+* Any crash, especially in Envoy
+* Any security policy bypass or weakness
+* Any potentional Denial Of Service (DoS)
```
Comment (ericvn) [inline content/en/about/security-vulnerabilities/index.md:31]: c/Of/of

Comment (fpesce) [inline content/en/about/security-vulnerabilities/index.md:31]:
```suggestion
* Any potential Denial of Service (DoS)
```

**Review by fpesce — CHANGES_REQUESTED** [review]

File: content/en/about/security-vulnerabilities/index.md [hunk 2]
```diff
@@ -25,6 +25,11 @@ Send us a report whenever you:
 - Think a vulnerability is present in another project that Istio
 depends on. For example, Envoy, Docker, or Kubernetes.
 
+When in doubt, please disclose privately. This includes, but is not limited to:
+* Any crash, especially in Envoy
+* Any security policy bypass or weakness
```
Comment (nrjpoddar) [inline content/en/about/security-vulnerabilities/index.md:30]:
```suggestion
* Any security policy (like Authentication or Authorization) bypass or weakness
```

**Review by ericvn — APPROVED** [review]

**Review by fpesce — APPROVED** [review]

**Pull request merged**